data/hai jobs.txt

```diff
@@ -396,6 +396,63 @@ mission "Delivery to Human Space [2]"
 		payment 80000
 		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
 
+mission "Hai Retrieve Human Luxury Goods"
+	name "Special order to <planet>"
+	job
+	repeat
+	description "A Hai socialite on <destination> wants to show off their knowledge of human culture to their peers. They will pay <payment> for <cargo> from <stopovers>."
```

Comment:
Question: Does `<destination>` or `<planet>` feel better here? I went with `<destination>` so that the return address would be in the description, but I want to hear what others think.

Reply:
It's a good idea to include some UI examples when asking for feedback like this

Reply:
Sure. 
Version with `<destination>`:
<img width="391" alt="destination description" src="https://user-images.githubusercontent.com/1037883/50580323-fd5d0a00-0e01-11e9-8f53-e981e8a261c1.png">
Version with `<planet>`:
<img width="388" alt="planet description" src="https://user-images.githubusercontent.com/1037883/50580325-0948cc00-0e02-11e9-881e-bcf5df3b7540.png">

Reply:
The `<planet>` version seems a bit more elegant to me.

Reply:
Switched to that.